server/api/region_test.go

```diff
@@ -331,3 +333,72 @@ func (s *testGetRegionSuite) TestScanRegionByKey(c *C) {
 		c.Assert(v, Equals, regions.Regions[i].ID)
 	}
 }
+
+// Create n regions (0..n) of n stores (0..n).
+// Each region contains np peers, the first peer is the leader.
+// (copied from server/cluster_test.go)
+func newTestRegions(n, np uint64) []*core.RegionInfo {
```

Comment:
np does not represent the number of stores, mostly used we use 3 for it. you can think it as one data have 3 replicas.

Reply:
Done